client/lib/media/test/actions.js

```diff
@@ -69,6 +69,7 @@ describe( 'MediaActions', function() {
 						return {
 							get: mediaGet.bind( [ siteId, mediaId ].join() ),
 							update: mediaUpdate.bind( [ siteId, mediaId ].join() ),
+							edit: mediaUpdate.bind( [ siteId, mediaId ].join() ),
```

Comment:
:D 

So, don't they do the same action? If yes, we should drop one here, IMO. Otherwise, I'll get confused. :D